manual/manual_circuit.rst

```diff
@@ -363,7 +364,44 @@ If one :py:class:`~pytket.circuit.Circuit` lacks some unit present in the other,
     next_circ.CZ(b[1], b[0])
 
     circ.append(next_circ)
-    circ
+    render_circuit_jupyter(circ)
+
+
+As a shorthand for the above we can take the tensor product of two circuits using the * operator. This requires that the qubits in the circuits have distinct names. 
+
+.. jupyter-execute::
+
+    from pytket import Circuit
+    from pytket.circuit.display import render_circuit_jupyter
+
+    circ1 = Circuit()
+    j = circ1.add_q_register("j", 1)
+    circ1.Y(j[0])
+
+    circ2 = Circuit()
+    k = circ2.add_q_register("k", 2)
+    circ2.X(k[1])
+    circ2.CRz(0.64, k[1], k[0])
+
+    circ3 = circ1 * circ2 # Take the tensor product
```

Comment:
Should we even cover the case for doing the tesnor product with append? This is described as an "extreme case" currently.

To me doing 

```python
circ1 * circ2 
```

is more natural 

Than doing

```python
circ1.append(circ2)
```

I would be in favour of recommending the `*` operator for composition vs append and mentioning `Circuit.append` as a note.

Maybe this is just because I'm not a big fan of `Circuit.append`. I'd much rather be explicit about where my operation/circuit is added by providing a list of unit ids with `Circuit.add_circuit` or `Circuit.add_gate`.

Reply:
I agree, we should introduce `append()` with an example where both circuits have the same set of unit IDs, and `*` with an example where they are disjoint.

Reply:
Nice, I've deleted the example where `Circuit.append` is used to take the tensor product. I've mentioned this behaviour as a note.

There was already an example where the two circuits shared unit ids which I have kept.